static/js/components/PersonalTab.js

```diff
@@ -35,32 +34,32 @@ class PersonalTab extends React.Component {
     ui:             UIState,
     nextStep:       () => void,
     prevStep:       () => void,
-    dashboard:      DashboardState,
+    programs:       ProgramEnrollmentsState,
     setProgram:     Function,
     addProgramEnrollment: Function,
   };
 
-  programListing = (programs: Array<Program>) => {
+  programListing = (programs: ProgramEnrollments) => {
     const makeMenuItems = R.map(program => (
       <MenuItem value={program.id} key={program.id} primaryText={program.title} />
     ));
     const sortPrograms = R.sortBy(R.compose(R.toLower, R.prop('title')));
     const menuItems = R.compose(makeMenuItems, sortPrograms);
-    return menuItems(programs) ;
-  }
+    return menuItems(programs);
+  };
 
   setProgramHelper = (event: Event, key: string, value: string) => {
     const {
-      dashboard: { programs },
+      programs: { programEnrollments },
```

Comment:
i think there are a couple problems here.

1. `programs` is a `*State` type. correct me if i'm wrong, but the general pattern elsewhere in our codebase is to handle these states with their associated fetch status/error text/etc in a container before passing the raw information down to the components (at least for `*State` types that represent a request for a complex object or array and not just a scalar value). maybe i'm missing something, but we don't seem to be doing anything with the extra state information. it seems like only the list of `ProgramEnrollments` should be passed down to this and other components.
1. speaking of which, the naming of `programEnrollments` is pretty confusing. this line from ProgramSelector.js kinda sums it up: `enrollment => enrollment.enrolled`. in english: "is this enrollment enrolled?". i think just calling this collection `programs` is more on the mark.